### ISSUE TITLE
MLGA only shows my ping

### ISSUE THREAD

**Issue by travLappy** [PsiLupan/MakeLobbiesGreatAgain#96]
When executing `java -jar MLGA.jar` I do not get any errors, but when I am in the lobby, it only shows my ping and not any of the other players.

**Comment by Josemaloco** [comment]
You are playing as a survivor right? In that case it only shows the killer's ping.

There is no such thing as showing your own ping since it would be 0. The ping is always a measure against another point in a network.

**Comment by travLappy** [comment]
Yes, you're right, I figured that out last night. Do the Blocked and Loved features not work anymore? I've tried doing as he says, but it only shows the killers ping.

**Comment by Josemaloco** [comment]
He removed that functionality on the latest patch [1.40](https://github.com/PsiLupan/MakeLobbiesGreatAgain/releases/tag/1.40) but 1.39 works the same as always. At least until some update breaks it.
